src/main/java/com/gmail/goosius/siegewar/hud/SiegeWarHud.java

```diff
@@ -1,5 +1,6 @@
 package com.gmail.goosius.siegewar.hud;
 
+import com.gmail.goosius.siegewar.objects.BattleSession;
 import com.gmail.goosius.siegewar.objects.Siege;
```

Comment:
This import also doesn't seem to be used.

Reply:
Fixed